test/integration-test.sh

```diff
@@ -280,104 +399,173 @@ log "hopr addr5: ${addr5}"
 # we don't need node6 because it's short-living
 log "hopr addr7: ${addr7}"
 
-log "Check peers"
-result=$(run_command ${api1} "peers" 'peers have announced themselves' 600)
-log "-- ${result}"
+test_withdraw() {
+  local node_api="${1}"
+
+  balances=$(get_balances ${node_api})
+  native_balance=$(echo ${balances} | jq -r .native)
+  hopr_balance=$(echo ${balances} | jq -r .hopr)
+
+  withdraw ${node_api} "NATIVE" 10 0x858aa354db6ae5ea1217c5018c90403bde94e09e
+  log "waiting 30 seconds for withdraw transaction to complete"
+  sleep 30
```

Comment:
We must avoid static wait windows because the test run-time will be increased. Instead if might make sense to split the test into two parts, and run the verification later.